roles/cert_verify/tasks/check_memcached.yml

```diff
@@ -0,0 +1,73 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Read memcached-0 config for tls
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command: "{{ lookup('env', 'HOME') }}/bin/oc exec -n openstack -c memcached -t memcached-0 -- bash -c 'cat /etc/sysconfig/memcached|grep OPTIONS'"
+  register: cert_verify_memcached_0_config
+
+- name: Check memcached-0 for tls key/cert
+  ansible.builtin.assert:
+    that:
+      - "'ssl_key=/etc/pki/tls/private/memcached.key' in cert_verify_memcached_0_config.stdout"
+      - "'ssl_chain_cert=/etc/pki/tls/certs/memcached.crt' in cert_verify_memcached_0_config.stdout"
+    success_msg: "memcached-0 is listening on SSL"
+    fail_msg: "memcached-0 is not SSL"
+
+- name: Check memcached-0 for tls serial
```

Comment:
imho you should create a task file to extract the serials and so on, and just `ansible.builtin.include_tasks` with parameters. The command is mostly the same, having a re-usable tasks file would allow to not copy-paste things so many time.
Thoughts?